source/js/main.js

```diff
@@ -83,154 +96,18 @@ let main = {
     envReq.send();
   },
 
-  fetchHomeDataIfNeeded(callback) {
```

Comment:
`fetchHomeDataIfNeeded` is super old code that hasn't been in use since almost two years ago 😂

Reply:
lol